atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/OkRead.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright 2016 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.jepsen;
+
+public class OkRead extends Event {
+    Long time;
+    Integer process;
+    Long value;
```

Comment:
These should be private too.
Are `time` and `value` required? If so I normally prefer to use the primitive long instead of the objects.